packages/fether-electron/src/main/index.js

```diff
@@ -57,12 +58,33 @@ function createWindow () {
         parityChannel: parity.channel
       })
     )
-    .then(() =>
+    .then(async () => {
       // Run parity when installed
-      runParity(['--light', '--chain', cli.chain || 'kovan'], err =>
-        handleError(err, 'An error occured with Parity.')
-      )
-    )
+
+      // Don't run parity if the user ran fether with --no-run-parity
+      if (!cli.runParity) {
+        return;
+      }
+
+      if (await isParityRunning({
+        wsInterface: cli.wsInterface,
+        wsPort: cli.wsPort
+      })) {
+        return;
+      }
+
+      return runParity({
+        flags: [
+          ...getRemainingArgs(cli),
+          '--light',
+          '--chain', cli.chain || 'kovan',
+          '--ws-interface', cli.wsInterface,
```

Comment:
This runs `parity --light --chain kovan --ws-interface --ws-port` if no --ws-{port|interface} is passed to fether, which then shows an error when running parity.

If there's a possibility to put a default value in commander.js, that'd be great. Or else I guess we'd need to do like above with --chain, i.e. `cli.wsInterface || 8546`.